pkg/syncer/metadatasyncer_test.go

```diff
@@ -760,3 +764,590 @@ func TestCalculateVMServiceStoragePolicyUsageReservedForNamespace(t *testing.T)
 		})
 	}
 }
+
+func TestSyncStorageQuotaReserved(t *testing.T) {
```

Comment:
can we add testcases here where negative value update skipped and pvc with same name but different size exists in different namespaces ?